.travis.yml

```diff
@@ -125,4 +125,11 @@ script:
       sh ./misc/travis/clang-format.sh;
     else
       scons -j2 CC=$CC CXX=$CXX platform=$PLATFORM tools=$TOOLS target=$TARGET $OPTIONS $EXTRA_ARGS;
+
+      if [ "$TEST_PROJECT" = "yes" ]; then
+        mkdir -p "$HOME/.local/share/godot/templates/3.2.dev/";
+        cp "bin/godot_server.x11.opt.tools.64" "$HOME/.local/share/godot/templates/3.2.dev/linux_x11_64_release";
+        git clone --depth 1 --branch add-test-project "https://github.com/Calinou/godot-tests.git";
```

Comment:
Don't merge yet, I'll change the repository URL and branch once https://github.com/godotengine/godot-tests/pull/7 is merged.

Reply:
You can rebase now.